src/controllers/taxonomias-controller.js

```diff
@@ -476,9 +475,8 @@ export const cadastrarGenero = (request, response, next) => {
 
             return familiaEncontrada;
         })
-        .then(familia => Genero.create({ nome,
+        .then(Genero.create({ nome,
```

Comment:
@MatCoitinho aqui você ainda precisa manter a função, mesmo que você não faça uso do parâmetro, no caso


```suggestion
        .then(() => Genero.create({ nome,
```